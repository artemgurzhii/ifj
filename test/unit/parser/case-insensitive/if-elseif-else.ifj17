iF 1 > 0 theN
  return 1
elSEif 2 > 1 tHEn
  return 2
eLSe
  return 3
eNd
