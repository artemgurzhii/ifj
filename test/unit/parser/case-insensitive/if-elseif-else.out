(if (> (int 1) (int 0))
  (return
    (int 1)))
(else if (> (int 2) (int 1))
  (return
    (int 2)))
(else
  (return
    (int 3)))
